### ISSUE TITLE
Add file filters support for macOS in lbg_choose_file()

### ISSUE THREAD

**Issue by pruje** [pruje/libbash.sh#10]
Transform *.extension to UTI format:
https://developer.apple.com/library/content/documentation/FileManagement/Conceptual/understanding_utis/understand_utis_intro/understand_utis_intro.html#//apple_ref/doc/uid/TP40001319

**Comment by pruje** [comment]
draft:
```bash
lbg_choosefile_opts=""

# prepare filters
if [ ${#lbg_choosefile_filters[@]} -gt 0 ] ; then
    lbg_choosefile_opts="{"

    for ((lbg_choosefile_i=0 ; lbg_choosefile_i < ${#lbg_choosefile_filters[@]} ; lbg_choosefile_i++)) ; do
        lbg_choosefile_filter="${lbg_choosefile_filters[$lbg_choosefile_i]}"

        # transform *.ext into UTI format
        echo "$lbg_choosefile_filter" | grep -E ^[*][.] &> /dev/null
        if [ $? == 0 ] ; then
            lbg_choosefile_filter="$(echo "$lbg_choosefile_filter" | sed 's/*/public/')"
        fi

        lbg_choosefile_opts+="$lbg_choosefile_filter,"
    done

    # delete last comma
    lbg_choosefile_opts="${lbg_choosefile_opts%?}}"
fi
```

**Comment by pruje** [comment]
Fixed in 65fe40f032b95e2585697cacf64ffc653819c49b
